runtime/src/bank.rs

```diff
@@ -3641,6 +3646,21 @@ impl Bank {
             .and_then(|instruction_recorder| Rc::try_unwrap(instruction_recorder).ok())
             .map(|instruction_recorder| instruction_recorder.into_inner().deconstruct());
 
+        let post_account_state_info =
+            self.get_transaction_account_state_info(&transaction_context, tx.message());
+
+        if self
+            .feature_set
+            .is_active(&feature_set::require_rent_exempt_accounts::id())
+        {
+            Self::verify_transaction_account_state_changes(
```

Comment:
It would be nice if the feature bool was passed in so that the metrics could still be submitted even when it's disabled.